gulp-tasks/reference-docs.js

```diff
@@ -0,0 +1,176 @@
+const gulp = require('gulp');
```

Comment:
Should this file be called workbox-reference-docs.js? or include comments near the top explaining what section this file is generating documentation for? Atm, Workbox team probably know what it does but I can see the rest of the team not.

Reply:
If it's going to be used for other projects later, we should keep the name, but if this is absolutely unique to workbox, then it should include something. It'd also be nice to have a small header with the appropriate description in it.

Reply:
I've tried to make it generic, although the gulp command is specific to workbox at the mo `gulp reference-docs:workbox`. So basically - no not intended to be specific, but only tested on workbox.